Packs/SecureWorks/Integrations/TaegisXDRv2/TaegisXDRv2.py

```diff
@@ -556,14 +560,14 @@ def fetch_alerts_command(client: Client, env: str, args=None):
 
 
 def fetch_assets_command(client: Client, env: str, args=None):
-    page = arg_to_number(args.get("page")) or 0
-    page_size = arg_to_number(args.get("page_size")) or 10
+    page = arg_to_number(args.get("page", 0))
```

Comment:
Hey, I'm sorry for the misunderstanding and the trouble, Didn't realize that was for preventing the linting and typing issues. I would be happy if you could revert to your previous code using the 'or 0' without the #type: ignore. 
Please apply this to all places with the type: ignore.

Reply:
I went ahead and reverted this for the 2 ignore lines.